app/plugin_api.go

```diff
@@ -369,6 +369,39 @@ func (api *PluginAPI) GetUserCustomStatus(userID string) (*model.CustomStatus, *
 	return api.app.GetCustomStatus(userID)
 }
 
+func (api *PluginAPI) RestoreToPreviousCustomStatus(userID string) *model.AppError {
```

Comment:
I'm thinking this method should be defined on `App`, and this plugin API method should just proxy to that method. Although the main use case for this is for plugins to call this method, the functionality in the function itself isn't really plugin-related.

Reply:
moved to `app`